### DISCUSSION TITLE
What Color Should Be Made Next?

### DISCUSSION THREAD

**Discussion by zelon88** [zelon88/HRConvert2#57] [Polls]
I intend on adding more color schemes to HRConvert2. I'm curious to see what colors the community would be interested in having!

**Comment by bit-man** [comment]
Color scheme for visually impaired people

**Comment by zelon88** [comment]
I think the way forward with high visibility is;

- Add "High Visibility" option to styleCore.php of all GUI's, plus an option to enable or disable it in config.php. Allow a naive GUI to gracefully ignore it without errors. 
- By adding "High Visibility" options to styleCore.php we are ensuring high vis users will have an overall better user experience. Instead of being shuffled to a potentially awkward UI, every UI will be able to adapt.

**Comment by zelon88** [comment]
Added 3 new color schemes.

Purple
Orange
Dark

Still no high vis.

**Comment by zelon88** [comment]
I'm going to close this, as the recent updates did do a lot for accessibility. 
If High Vis comes up again we can revisit this.
